src/core/api/index.js

```diff
@@ -170,3 +170,13 @@ export function categories({ api }) {
     state: api,
   });
 }
+
+export function logOutFromServer({ api }) {
+  return callApi({
+    auth: true,
+    credentials: true,
+    endpoint: 'accounts/session',
+    method: 'delete',
+    state: api,
+  });
```

Comment:
does it make any sense to check the result to make sure `ok` is true? Or maybe a better question: will there be any `200` responses to this request that indicate a failure?